openapi/backend/openapi.yaml

```diff
@@ -1231,12 +1231,15 @@ components:
         name:
           type: string
           minLength: 1
+          maxLength: 32
```

Comment:
Do the generated openapi objects check these limits before it gets to your code?

Reply:
Unfortunately, the library webstatus uses does not do the validation.
---
On a semi related note, the library used in chrome status [does](https://github.com/OpenAPITools/openapi-generator/blob/0e34d398337dbeea8136bbdf830cdecb3110f618/modules/openapi-generator/src/main/resources/python-flask/model.mustache#L132).